Formula/gpgme.rb

```diff
@@ -12,6 +12,8 @@ class Gpgme < Formula
     sha256 "4a4c3e90175b02ded974a8a385b6d9799bcc2873c5a1087c558f350eb103934d" => :yosemite
   end
 
+  patch :DATA
```

Comment:
Sorry it's been taking me so long to get back to this... I'm half thinking that it might be worth waiting for a gpgme 1.8.1?